src/vivarium/framework/state_machine.py

```diff
@@ -199,12 +214,18 @@ def model(self) -> str:
     # Lifecycle methods #
     #####################
 
-    def __init__(self, state_id: str, allow_self_transition: bool = False):
+    def __init__(
+        self,
+        state_id: str,
+        allow_self_transition: bool = False,
+        initialization_weights: Callable[[Builder], LookupTableData] = default_initializer,
```

Comment:
nit: I think `callable` is a built-in now?

Reply:
`callable(foo)` is a function that returns True if `foo` is callable. It's not a something that can be used in typing.